src/Server/Services/Jobs/DataRetrievalService.cs

```diff
@@ -96,7 +97,7 @@ private async Task BackgroundProcessing(CancellationToken cancellationToken)
                 try
                 {
                     request = await _inferenceRequestStore.Take(cancellationToken);
-                    using (_logger.BeginScope(new Dictionary<string, object> { { "JobId", request.JobId }, { "TransactionId", request.TransactionId } }))
+                    using (_logger.BeginScope(new LogginDataDictionary<string, object> { { "JobId", request.JobId }, { "TransactionId", request.TransactionId } }))
```

Comment:
what is the comparer used by this dictionary? default string comparer is `CurrentCultureComparer` which is almost always the wrong comparer.

Reply:
No comparison required here; this is to ensure these keys/values are logged.